platform/viewer/cypress/integration/common/OHIFRoutes.spec.js

```diff
@@ -0,0 +1,42 @@
+// type definitions for Cypress object "cy"
+/// <reference types="cypress" />
+
+describe('OHIF Routes', function() {
+  beforeEach(function() {
+    cy.openStudyList();
+  });
+
+  it('checks PT/CT json url study route', function() {
+    cy.visit(
+      '/viewer?url=https://s3.eu-central-1.amazonaws.com/ohif-viewer/JSON/PTCTStudy.json'
+    );
+    cy.server();
+    cy.route('GET', '**/PTCTStudy/**').as('getPTCTStudy');
+
+    cy.wait('@getPTCTStudy.all');
+    cy.get('@getPTCTStudy').should($route => {
+      expect($route.status).to.be.eq(200);
+    });
```

Comment:
I tried to think about how I might test this, and I can't think of anything more resilient, except maybe creating a `json` file that's part of our app we use specifically for this test (instead of an external resource).

Can we drop a PercySnapshot in here? Probably after a minimum thumbnail load or something similar?

Reply:
Hmm. Not sure exactly how we can navigate/load the local `json` file. The following code isn't working, any thoughts?

```
cy.fixture('./../fixtures/PTCTStudy.json').as('PTCTStudyJSON');

cy.visit('/');
cy.request('viewer?url=PTCTStudyJSON');
```